wabt-sys/build.rs

```diff
@@ -88,6 +88,18 @@ git submodule update --init --recursive",
         cfg.flag("-std=c++17");
     }
 
+    // Generally, workaround for https://github.com/rust-lang/cc-rs/pull/506
+    if target_os == "windows" {
+        // CMake links debug or release C runtime when `cc` crate links only release one.
+        // So we just set `_DEBUG` flag for symbols to be the same
+        if env::var("DEBUG").ok().as_deref() == Some("true") {
+            cfg.define("_DEBUG", "1");
+        }
+
+        // `cc` crate tracks target features when CMake just builds with dynamic C runtime
+        cfg.static_crt(false);
```

Comment:
IMHO it is better to configure CMake accordingly. If I'm not mistaken build type should be set explicitly via https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html since the default value is empty string. `RelWithDebInfo`/`Release` should be a good choice.

For selecting static/dynamic runtime library the config variable https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html can be used.

Reply:
> If I'm not mistaken build type should be set explicitly via [cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html)

`cmake` crate already handles it

> the config variable [cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html](https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html) can be used.

Done